src/main/twirl/com/lightbend/lagom/docs/sideNav.scala.html

```diff
@@ -4,16 +4,16 @@
 
 <h3>Documentation</h3>
 <ul>
-    <li><a href="@context.path/get-started.html">Try Lagom</a></li>
-    <li><a href="@context.path/documentation/">Full documentation</a></li>
-    <li><a href="@context.path/faq.html">FAQ</a></li>
-    <li><a href="@context.path/blog">Blog</a></li>
+    <li><a href="@context.path/documentation">By release</a></li>
+    <li><a href="@context.path/get-started.html">Get started</a></li>
+    <li><a href="@context.path/faq.html">FAQs</a></li>
```

Comment:
Also "FAQs" vs. "FAQ" is now inconsistent between the side navigation and the top navigation.

Reply:
I can change that back.

Reply:
Done. Will commit soon.